src/NeoLocalPlanner.cpp

```diff
@@ -167,6 +167,53 @@ NeoLocalPlanner::~NeoLocalPlanner()
 {
 }
 
+void NeoLocalPlanner::reconfigureCB(NeoLocalPlannerConfig& config,uint32_t level) {
```

Comment:
It would be better to add the `odometery_mutex` here as well..

Reply:
I don't know why you need a mutex around just the odometry here. I don't think there are any odometry-related variables in the dynamic reconfigure callback.